JASP-Engine/JASP/R/LSbinomialcommon.R

```diff
@@ -297,7 +298,7 @@
       text_Alpha <- MASS::fractions(prior[["parAlpha"]] + data$nSuccesses)
     }
     if (!is.na(as.numeric(prior[["parBeta_inp"]]))){
-      text_Beta <- prior[["parBeta"]] + data$nSuccesses
+      text_Beta <- prior[["parBeta"]] + data$nFailures
```

Comment:
Is data here a specific "jasp" object? Otherwise, it might be good to change this to [["nFailures"]]

Reply:
yes, it's a list, might be accessed with [[]], changed